lib/src/util/react_util.dart

```diff
@@ -57,6 +63,14 @@ class UiPropsMapView extends MapView
   void modifyProps(PropsModifier modifier, [bool shouldModify = true]) =>
       throw UnimplementedError('@PropsMixin instances do not implement modifyProps');
 
+  @override
+  void addUnconsumedProps(Map props, Iterable<PropsMeta> consumedProps) =>
+      throw UnimplementedError('@PropsMixin instances do not implement addUnconsumedProps');
+
+  @override
+  void addUnconsumedDomProps(Map props, Iterable<PropsMeta> consumedProps) =>
+      throw UnimplementedError('@PropsMixin instances do not implement addUnconsumedDomProps');
+
```

Comment:
```suggestion
```

Same comment as above

Reply:
The `StrictMode` component was as expected, but I'm seeing it as necessary here. And we have the other members (`addProps`) overridden here. Did I miss something though?

Reply:
Ah, right; sorry about that!

Reply:
No worries!!